### ISSUE TITLE
Socks5 config with different servers but identical port/username/password problem.

### ISSUE THREAD

**Issue by macpuffer** [FelisCatus/SwitchyOmega#2184]
Hi there, first of all, I do love the switcher and the gui, well done, only thing is that it hides the favorites bar, but it's a minor problem!
I use a paid subscription from Privado, where I can choose from many socks5 servers all of which share the same port/username and password, only the server changes. So, when I add the first proxy it works just fine, and when I add the second one, that one works as well but if I switch back to the first one, it would give a 'unable to connect' for any website I open and also hide the favorites bar. This will continue to happen for all new profiles I enter. Only the last saved one works. Upon closing/opening FirefoxNightly it is a matter of luck which of the profiles work and which not, but most do not and sometimes none does.
It also doesn't help to keep just one single profile and change the server address as needed, because when you do that the once working profile stops doing so.
In short: Socks5 with authentication works only with one freshly saved profile.
I do hope I'm making sense.

Windows 10 Pro 20H2
Firefox nightly 90.0a1 (2021-05-11) (64-bit)

SwitchyOmega 2.5.20
Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:90.0) Gecko/20100101 Firefox/90.0

**Comment by macpuffer** [comment]
Well ok, please ignore the mentioning of a favorites bar problem, I didn't know it's a feature of Firefox and certainly not a problem of Switchy.
